### ISSUE TITLE
parallelization levels

### ISSUE THREAD

**Issue by berndbischl** [mlr-org/mlr-tutorial#93]
1) the new mlr.ensemble level is not documented on the parallel page, basic section

2) we now have the same docs twice. it is also in parallel.R.
just link to that, so the peoblem 1) does not happen in the future

REALLY simple to fix, do it now

**Comment by schiffner** [comment]
Duplicate of #73
